src/box/lua/tuple.c

```diff
@@ -81,6 +90,15 @@ uint32_t CTID_STRUCT_TUPLE_REF;
  */
 static int luaT_tuple_encode_table_ref = LUA_NOREF;
 
+static int luaT_tuple_format_iterator_next_ref = LUA_NOREF;
+
+/**
+ * If set, `box_tuple_new` will interpret an argument list (i.e., more than one
+ * argument passed) as a list of tuple fields.
+ */
+bool box_tuple_new_arg_list;
```

Comment:
This should be `static`.

Reply:
Indeed, fixed. I thought it needs to be extern, since it's a tweak option.